01_Core/week03/submissions/exercise 7/index.html

```diff
@@ -0,0 +1,163 @@
+<!doctype html>
```

Comment:
There's a lot of repetition in your styles. Try to refactor using opacity and a single color for each div